### PR TITLE
Fix deploy-storybook

### PR DESCRIPTION
**Pull request by jrschumacher**
Fix deploy storybook

### CODE REVIEW

File: .github/workflows/deploy-storybook.yml [hunk 1]
```diff
@@ -35,10 +35,10 @@ jobs:
 
       - name: Create Release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{github.actor}}.${{ secrets.GITHUB_TOKEN }}
```
Comment (jrschumacher) [inline .github/workflows/deploy-storybook.yml:38]:
```suggestion
          GITHUB_TOKEN: ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/deploy-storybook.yml [hunk 2]
```diff
@@ -35,10 +35,10 @@ jobs:
 
       - name: Create Release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}
```
Comment (avkvirtru) [inline .github/workflows/deploy-storybook.yml:38]: FYI on [the secrets page](https://github.com/virtru/virtuoso-design-system/settings/secrets), this is `GH_TOKEN`

Comment (jrschumacher) [inline .github/workflows/deploy-storybook.yml:38]:
```suggestion
          GITHUB_TOKEN: ${{ github.actor }}:${{ secrets.GH_TOKEN }}
```

**Comment by avkvirtru** [comment]
:rocket: PR was released in [`v1.6.7`](https://github.com/virtru/virtuoso-design-system/releases/tag/v1.6.7) :rocket:

**Pull request merged**